prow/cmd/peribolos/main.go

```diff
@@ -216,8 +216,8 @@ type dumpClient interface {
 	GetOrg(name string) (*github.Organization, error)
 	ListOrgMembers(org, role string) ([]github.TeamMember, error)
 	ListTeams(org string) ([]github.Team, error)
-	ListTeamMembers(org string, id int, role string) ([]github.TeamMember, error)
-	ListTeamRepos(org string, id int) ([]github.Repo, error)
+	ListTeamMembersBySlug(org, teamSlug, role string) ([]github.TeamMember, error)
```

Comment:
This is a purely internal change, right? We never read a teamid from the config, we only got them through the `ListTeams` api call?

Reply:
yes, the usages both come from [line 276](https://github.com/kubernetes/test-infra/blob/3686760017b566c041256a04026619f93196b917/prow/cmd/peribolos/main.go#L276) and